flopy/modflow/mfhfb.py

```diff
@@ -195,18 +197,30 @@ def write_file(self):
         for option in self.options:
             f_hfb.write(f"  {option}")
         f_hfb.write("\n")
+
+        openclose = False
```

Comment:
should this be set in the condition below, otherwise it is always false and the file is never closed

Reply:
good catch. Thanks